var/spack/repos/builtin/packages/serenity/package.py

```diff
@@ -40,15 +50,17 @@ class Serenity(CMakePackage):
 
     extends("python", when="+python")
 
+    patch("1.6.1-cmake.patch", when="@1.6.1:")
+
     def patch(self):
-        filter_file(
-            "include(CMakeParseArguments)",
-            'message(FATAL_ERROR "Tried to download a dependency")',
-            "cmake/DownloadProject.cmake",
-            string=True,
-        )
-
-        if self.run_tests:
+        if self.spec.satisfies("@1.4.0"):
```

Comment:
These devendoring steps should be kept for other versions as well, since too many packages which can be provided and configured by spack are downloaded in the build process

Reply:
Ok, to be safe I kept the first one that throws an error if a package is downloaded by Cmake.
The others are not needed anymore, as the [Cmake file](https://github.com/qcserenity/serenity/blob/master/CMakeLists.txt) already include those since version 1.5.0, and I explicitly deactivate downloads with https://github.com/spack/spack/blob/e275b5e757d092d814d78a2f72891cb3651472d0/var/spack/repos/builtin/packages/serenity/package.py#L166

Reply:
@awvwgk Thanks!